plutus-tx/testlib/PlutusTx/Test.hs

```diff
@@ -139,8 +144,7 @@ goldenBundle
 goldenBundle name x y = do
   goldenPirReadable name x
   goldenUPlcReadable name x
-  goldenEvalCekCatch name [y]
-  goldenBudget name y
```

Comment:
Do we even `goldenEvalCekCatch` and `goldenBudget` anymore? If not, let's remove them (or one of them, if we need the other) to ensure only `goldenEvalCekCatchBudget` gets used from now on.

Reply:
Okay, I'll count this as a greenlight to kill them all. I was little reluctant to just take out functions